pandas-stubs/core/arrays/numpy_.pyi

```diff
@@ -62,6 +63,7 @@ class PandasArray(ExtensionArray, ExtensionOpsMixin, NDArrayOperatorsMixin):
     ): ...
     def prod(
         self,
+        *,
```

Comment:
Many methods in PandasArray are keyword-only but some sub-classes allow also positional arguments for them.

Reply:
I have to ask if we should be including `PandasArray` in the stubs.  While the class is documented, none of its methods are documented.

Reply:
If none of its methods are documented to be public, it seems reasonable to remove the entire class. For now, I wouldn't want to do that since many sub-classes do not "overwrite" its methods in pandas-stubs (min/max/mean/median/sum/...).

Reply:
I removed all methods except one which needs to be "implement", because it is defined as abstract in a parent class.

Reply:
thanks